packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -75,54 +53,31 @@ export const CallControls = (props: CallControlsProps): JSX.Element => {
             dockedBottom it has position absolute and would therefore float on top of the media gallery,
             occluding some of its content.
          */}
-        <ControlBar layout="horizontal" styles={props.controlBarStyles}>
+        <ControlBar layout="horizontal">
           {customButtons['first']}
-          {options?.microphoneButton !== false && (
-            <Microphone
-              displayType={options?.displayType}
-              styles={props.commonButtonStyles}
-              splitButtonsForDeviceSelection={props.splitButtonsForDeviceSelection}
-            />
-          )}
+          {isEnabled(options?.microphoneButton) && <Microphone displayType={options?.displayType} />}
```

Comment:
where did all the props go for these buttons?

Reply:
... magic ... :P

The `<Microphone>` component calls for `usePropsFor(MicrophoneButton)`. This allows us to encapsulate button specific logic in these wrappers, and makes the code flow here linear -- the only thing happening here is putting the buttons in the right sequence.